arroyo-api/src/sources.rs

```diff
@@ -196,6 +196,21 @@ pub enum PrimitiveType {
     UnixMillis,
 }
 
+fn primitive_to_sql(primitive_type: &PrimitiveType) -> &'static str {
+    match primitive_type {
+        PrimitiveType::Int32 => "INTEGER",
+        PrimitiveType::Int64 => "BIGINT",
+        PrimitiveType::UInt32 => "UNSIGNED INTEGER", // not really sql
+        PrimitiveType::UInt64 => "UNSIGNED BIGINT",  // not really sql
```

Comment:
These types don't have a sql equivalent, right?

Reply:
I'd look at https://arrow.apache.org/datafusion/user-guide/sql/data_types.html. In which case `INTEGER UNSIGNED` and `BIGINT UNSIGNED`.